void's numpy uses a different path, and it doesn't seem to be possible to set it on Linux

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2519,7 +2519,7 @@
     # Always use numpy bundled in precompiled libs.
   elseif((WITH_PYTHON_INSTALL AND WITH_PYTHON_INSTALL_NUMPY) OR WITH_PYTHON_NUMPY)
     if(("${PYTHON_NUMPY_PATH}" STREQUAL "") OR (${PYTHON_NUMPY_PATH} MATCHES NOTFOUND))
-      find_python_package(numpy "core/include")
+      find_python_package(numpy "_core/include")
     endif()
   endif()
 
